[INFO -- 2018-12-08 16:40:31]
Starting flip_value script
version: 1.1
[INFO -- 2018-12-08 16:40:31]
Initialization strings:
[INFO -- 2018-12-08 16:40:31]
(1) file /tmp/knapsack/sa
[INFO -- 2018-12-08 16:40:31]
initSignal:0
[INFO -- 2018-12-08 16:40:31]
endSignal:9
[INFO -- 2018-12-08 16:40:31]
faultModel:2
[EXCEPTION -- 2018-12-08 16:40:34]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-12-08 16:40:35]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-12-08 16:40:38]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-12-08 16:40:39]
gdbException: Cannot access memory at address 0x4094f1f257fd18cc
[INFO -- 2018-12-08 16:40:39]
Backtrace BEGIN:
#0  0x0000555555555861 in std::vector<int, std::allocator<int> >::operator[] (this=0x555555759060 <val2>, __n=180) at /usr/include/c++/7/bits/stl_vector.h:798
#1  0x00005555555551dd in cost (solution=std::vector of length -1163379128286673787, capacity -1163379128286673787 = {...}) at knapsacksa.cpp:63
#2  0x000055555555534c in simulated_annealing () at knapsacksa.cpp:92
#3  0x0000555555555658 in main () at knapsacksa.cpp:146

793	       */
794	      reference
795	      operator[](size_type __n) _GLIBCXX_NOEXCEPT
796	      {
797		__glibcxx_requires_subscript(__n);
798		return *(this->_M_impl._M_start + __n);
799	      }
800	
801	      /**
802	       *  @brief  Subscript access to the data contained in the %vector.

Backtrace END
Memory content before bitflip:0xc80x000x000x00
Fault Model: Random bit-flip
base address to flip value: 0x555555759038 
address max offset: 4
Memory content after  bitflip:0xb70xa70xfe0x5f
frame name: simulated_annealing
symbol name: n
symbol filename: knapsacksa.cpp
symbol line: 13
value: 1610524599
value address: 0x555555759038 <n>
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2018-12-08 16:40:39]
Fault Injection Successful after 8.554105997085571s
[INFO -- 2018-12-08 16:40:39]
event type: exit
[EXCEPTION -- 2018-12-08 16:40:39]
exit code: no exit code available
